src/actions/appActions.js

```diff
@@ -192,6 +193,9 @@ export const initAppAndRedirectAction = (appState: string, platform: string) =>
       const { insights = {} } = await storage.get('insights');
       dispatch({ type: SET_INSIGHTS_STATE, payload: insights });
 
+      const { hasSeenExchangeIntro = {} } = await storage.get('hasSeenExchangeIntro');
```

Comment:
What do you think about re-using appSettings for that?
I assume we're going to have more screens those should be seen only once, so it's better to re-use some existing storage table rather than flooding the new ones

Reply:
We have insights, userSettings and appSettings those could be re-used

Reply:
yeah... and those intro modals were intended for this very reason ;/
I guess there was a huge miscommunication when assigning the tasks.
Jan, we can discuss it in slack if you like. We have UI for those modals and a way to store the state of seen or not seen already